src/pages/documentation/media-for-edge-network/index.md

```diff
@@ -0,0 +1,78 @@
+import Tabs from './tabs/index.md'
+
+# Adobe Edge Media Analytics for Audio and Video
+
+<InlineAlert variant="info" slots="text"/>
+
+This extension requires the setting up of Datastreams with Media forwarding enabled, Dataset, XDM Schema and other AEP settings. Please follow this `backendDocumentationLink` before configuring and implementing the SDK.  
+
+
+## Configure Edge Media Analytics extension in the Data Collection UI
+
+<InlineAlert variant="info" slots="text"/>
+
+Media for Edge Network depends on Edge and Edge Identity extensions. Make sure to [configure the Edge extension in Data Collection UI](https://developer.adobe.com/client-sdks/documentation/edge-network/#configure-the-edge-network-extension-in-data-collection-ui) and [configure the Edge Identity extension in Data Collection UI](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/#configure-the-identity-extension-in-the-data-collection-ui) before proceeding.
+
+1. In the Data Collection UI, select the **Extensions** tab.
+2. On the **Catalog** tab, locate the **Adobe Edge Media Analytics for Audio and Video** extension, and select **Install**.
+3. Type the extension settings. For more information, see [Configure Media Analytics Extension](#configure-media-analytics-extension).
+4. Select **Save**.
+5. Follow the publishing process to update your SDK configuration.
+
+## Configure the Edge Media Analytics extension
+
+<InlineAlert variant="info" slots="text"/>
+
+![Adobe Edge Media Analytics Extension Configuration](./assets/index/configuration.png)
+
+To configure the Edge Media Analytics extension, complete the following steps:
+
+### Channel
+
+Type the channel name property.
+
+### Player name
+
+Type the name of the media player in use (for example, _AVPlayer_, _Native Player_, or _Custom Player_).
+
+### Application version
+
+Type the version of the media player application/SDK.
+
+## Add Edge Media Analytics to your app
+
+<InlineAlert variant="info" slots="text"/>
+
+This extension requires the [Adobe Analytics extension](../adobe-analytics/index.md). You must add the Analytics extension to your mobile property and make sure the extension is correctly configured.
```

Comment:
This isn't accurate as the Analytics extension is not required. However, you can add a note stating the Edge and Edge Identity extensions are required.